visualization/dashboard/src/MLIDashboard/Controls/AxisConfigurationDialog/AxisConfigDialog.tsx

```diff
@@ -148,8 +148,9 @@ export class AxisConfigDialog extends React.PureComponent<IAxisConfigProps, IAxi
                         )}
                         {!selectedMeta.treatAsCategorical && (
                             <div>
-                                <Text variant={"small"} className={styles.featureText}>
-                                {`Min: ${selectedMeta.featureRange.min}`} {`Max: ${selectedMeta.featureRange.max}`}
+                                <Text variant={"small"} className={styles.featureText} nowrap block>
+                                {localization.Filters.min}{(selectedMeta.featureRange.min % 1) !=0 ? (Math.round(selectedMeta.featureRange.min * 10000) / 10000).toFixed(4): selectedMeta.featureRange.min}{localization.Filters.space}
```

Comment:
cool, this makes sense to have a hardcoded space, but it may be easier to read as a template literal https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals
eg: `${localization.Filters.min}${<calculate the min string and max string earlier and set to variable used here>} <space> ${localization.Filters.max}${max string}